django-stubs/core/serializers/python.pyi

```diff
@@ -1,14 +1,13 @@
-from collections.abc import Iterator
 from typing import Any
 
 from django.core.serializers import base
-from django.core.serializers.base import DeserializedObject
 from django.db.models.base import Model
 
 class Serializer(base.Serializer):
     objects: list[Any]
     def get_dump_object(self, obj: Model) -> dict[str, Any]: ...
 
-def Deserializer(
-    object_list: list[dict[str, Any]], *, using: str = "default", ignorenonexistent: bool = False, **options: Any
-) -> Iterator[DeserializedObject]: ...
+class Deserializer(base.Deserializer):
+    def __init__(
+        self, object_list: list[dict[str, Any]], *, using: str = ..., ignorenonexistent: bool = False, **options: Any
```

Comment:
It can be any `Iterable`, not just `list`: https://github.com/django/django/blob/1831f7733d3ef03d1ca7fac3e8d9f4c5e3e3375e/django/core/serializers/python.py#L121-L123